src/utils/maybe-utils.js

```diff
@@ -36,3 +36,7 @@ export const isNone = m => m.isNone();
 export const orElseRun = R.curry((fn, m) => m.orElseRun(fn));
 
 export const isMaybe = any => Object.getPrototypeOf(any) === Maybe.prototype;
+
+export const findById = R.curry((id, collection) =>
+  R.compose(Maybe.fromNull, R.find(R.propEq('id', id)))(collection)
+);
```

Comment:
This is already done in a separate branch where koodisto-handling is handed over to dedicated koodisto-utils.

Reply:
this maybe even more generic function